packages/containers/src/List/List.connect.js

```diff
@@ -66,6 +77,10 @@ export function mergeProps(stateProps, dispatchProps, ownProps) {
 
 export default cmfConnect({
 	defaultState: DEFAULT_STATE,
+	defaultProps: {
+		saga: 'List#root',
```

Comment:
`saga: true`

Reply:
we should provide saga Id to mount the saga